docs/code-samples/getting-started/daml.yaml.template

```diff
@@ -49,5 +49,3 @@ data-dependencies:
 start-navigator: no
 build-options:
   - --target=1.15
-sandbox-options:
```

Comment:
Strange.. when I run the `daml new ...` command using SDK 2.5 I don't get this option in the daml.yaml 

So I'm wondering, do we have this differently in the `daml` repo, or is this daml.yaml.template file used differently than how I think?

Reply:
@matteolimberto-da This is the change we did earlier today. Do you know more about how it is used?

As far as I know, this file is used to generate the daml.yaml for the quickstarter in the `daml` repo. The `sandbox-options` were not set in the `daml` repo though: https://github.com/digital-asset/daml/pull/16357/files
(only in the `daml-finance` repo, and the files are supposed to be kept identical).